README.md

```diff
@@ -2,79 +2,3 @@
 
 This is a collection of tools used for Cloud Samples maintenance and infrastructure.
 
-This tool has one function:
```

Comment:
If this part of the main README is being moved to ariwete/README.md, there should probably be a link to that. (Presumption: ariwete is one of the (presumably) multiple tools in this cloud-samples-tools repo)

This main README should also include the standard README elements for OSS repos (Contributing/license, etc)

Reply:
Definitely agreed. And yes, I'm moving this into a sub directory with the idea that there may be other tools in the future. I was planning to follow up with a separate PR for the top level README. Seem reasonable?

Reply:
That seems fine. 

As long as the renames here lead to a successful build (based on the CI / Test check succeeding, but not validating the check itself), then I'll approve this. We can always fix things in future PRs.